<ui version="4.0" >
 <author>Sebastian Pipping</author>
 <class>toSearchReplaceUI</class>
 <widget class="QWidget" name="toSearchReplaceUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>122</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <property name="horizontalSpacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>F&amp;ind:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>SearchText</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="SearchText" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Text to search for</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="maxCount" >
        <number>30</number>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::InsertAtTop</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="SearchNext" >
       <property name="toolTip" >
        <string>Jump to next match</string>
       </property>
       <property name="text" >
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="SearchPrevious" >
       <property name="toolTip" >
        <string>Jump to previous match</string>
       </property>
       <property name="text" >
        <string>&amp;Previous</string>
       </property>
       <property name="icon" >
        <iconset>
         <normaloff>../../../</normaloff>../../../</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Rep&amp;lace:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy" >
        <cstring>ReplaceText</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="ReplaceText" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Text to replace with</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="Replace" >
       <property name="toolTip" >
        <string>Replace next match</string>
       </property>
       <property name="text" >
        <string>&amp;Replace</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QPushButton" name="ReplaceAll" >
       <property name="toolTip" >
        <string>Replace all matches</string>
       </property>
       <property name="text" >
        <string>Replace &amp;All</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4" >
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>6</number>
       </property>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="SearchMode" >
         <property name="toolTip" >
          <string>Search mode</string>
         </property>
         <property name="maxVisibleItems" >
          <number>4</number>
         </property>
         <property name="frame" >
          <bool>true</bool>
         </property>
         <item>
          <property name="text" >
           <string>Plain Text</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Regular Expression</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>6</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="MatchCase" >
         <property name="toolTip" >
          <string>Case-sensitive searching</string>
         </property>
         <property name="text" >
          <string>&amp;Match case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="WholeWords" >
         <property name="text" >
          <string>Whole words</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QToolButton" name="hideButton" >
       <property name="text" >
        <string/>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SearchText</tabstop>
  <tabstop>SearchNext</tabstop>
  <tabstop>SearchPrevious</tabstop>
  <tabstop>ReplaceText</tabstop>
  <tabstop>Replace</tabstop>
  <tabstop>ReplaceAll</tabstop>
  <tabstop>SearchMode</tabstop>
  <tabstop>MatchCase</tabstop>
  <tabstop>hideButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
